IOS/SwiftSeedProject/SwiftSeedProject/Interfaces/ViewControllers/NewsFeedCollectionViewController.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  NewsFeedCollectionViewController.swift
+//  SwiftSeedProject
+//
+//  Created by Lucas Pelizza on 4/21/17.
+//  Copyright © 2017 Making Sense. All rights reserved.
+//
+
+
+import UIKit
+import Bond
+import Dip_UI
+import Dip
+
+private let reuseIdentifier = "NewsCollectionViewCell"
+
+class NewsFeedCollectionViewController: UICollectionViewController {
+
+    var viewModel: NewsFeedViewModel!
+    var dataSource: MutableObservableArray<MutableObservableArray<NewsArticleViewModel>>!
```

Comment:
Why is it a MutableObservableArray of MutableObservableArrays? Couldn't you bind viewModel.newsArticles directly?